client/my-sites/plan-features/index.jsx

```diff
@@ -832,11 +834,14 @@ export default connect(
 
 				// Show price divided by 12? Only for non JP plans, or if plan is only available yearly.
 				const showMonthlyPrice = ! isJetpack || isSiteAT || ( ! relatedMonthlyPlan && showMonthly );
+				let features = planConstantObj.getPlanCompareFeatures( abtest );
 
-				let planFeatures = getPlanFeaturesObject(
-					planConstantObj.getPlanCompareFeatures( abtest )
-				);
+				// TODO: remove this once Quick Start sessions have been removed from Business Plan
+				if ( PLAN_BUSINESS === plan ) {
```

Comment:
It might be safer to use [isWpComBusinessPlan()](https://github.com/Automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/client/lib/plans/index.js#L241) rather than comparing against the constant since there are also constants for the 2 year and 1 month term versions of the plan. Similar to how `isWpComEcommercePlan()` is [used in this same file](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plan-features/index.jsx#L874).